tests/config/issue-2794.toml

```diff
@@ -0,0 +1,3 @@
+indent_style = "Block"
```

Comment:
You can put these config options inline using `rustfmt` comments, see some of the other tests for example